adders/bootstrap/config/tests.js

```diff
@@ -51,14 +51,14 @@ export const tests = defineAdderTests({
         {
             name: "typography properties",
             run: async ({ expectProperty }) => {
-                await expectProperty(".h1", "font-size", "34px");
+                await expectProperty(".h1", "font-size", "40px");
```

Comment:
the default viewport size of puppeteer is 800x600, while playwright's is 1280x720. given that `.h1` uses `rem` to for the font-size, this value needed to be tweaked. 

thankfully it was the only test that needed adjusting in this regard